app/Http/Controllers/Api/MovieController.php

```diff
@@ -15,7 +16,13 @@ class MovieController extends Controller
      */
     public function index()
     {
-        return Movie::all();
+        $user = JWTAuth::parseToken()->authenticate();
```

Comment:
in case that user is not authorized to view list of films we should return some error